test/ui-test/testSuites/suite_status/tst_ChatFlow/test.feature

```diff
@@ -50,6 +50,14 @@ Feature: Status Desktop Chat
          Then the user can delete the message at index 0
          Then the last message is not the random message
 
+    Scenario: User can send a gif
+         When the user opens app settings screen
+         When the user opens the messaging settings
+         When the user activates link preview
+         When the user opens the chat section
+         When user joins chat room automation-test
```

Comment:
Change all the When here except the first one to `And` to it makes more sense when reading.